src/wad/level.rs

```diff
@@ -149,12 +147,14 @@ impl Level {
         let mut min_light = sector.light;
         let sector_id = self.sector_id(sector);
         for line in self.linedefs.iter() {
-            let (left, right) =
-                (match self.left_sidedef(line) {
-                    Some(l) => l.sector, None => continue,
-                 }, match self.right_sidedef(line) {
-                    Some(r) => r.sector, None => continue,
-                 });
+            let (left, right) = (match self.left_sidedef(line) {
+                Some(l) => l.sector,
+                None => continue,
+            },
+                                 match self.right_sidedef(line) {
+                Some(r) => r.sector,
+                None => continue,
+            });
```

Comment:
The indentation on Some & None seems completely wrong, could you please fix this?